clang/docs/UndefinedBehaviorSanitizer.rst

```diff
@@ -293,6 +293,40 @@ To silence reports from unsigned integer overflow, you can set
 ``-fsanitize-recover=unsigned-integer-overflow``, is particularly useful for
 providing fuzzing signal without blowing up logs.
 
+Disabling instrumentation for common overflow patterns
+------------------------------------------------------
+
+There are certain overflow-dependent or overflow-prone code patterns which
+produce a lot of noise for integer overflow/truncation sanitizers. To disable
+instrumentation for these common patterns one should use
+``-fsanitize-overflow-pattern-exclusion=``.
+
+Currently, this option supports three pervasive overflow-dependent code idioms:
```

Comment:
It might be useful to add "titles" to each example here. Something like:

```txt
``negated-unsigned-const``

.. code-block:: c++

    /// -fsanitize-overflow-pattern-exclusion=negated-unsigned-const
 ...
```

Reply:
thanks. resolved in 4b3efbb41ff86eeff15671b1d876e1ef6a58a536